.travis.yml

```diff
@@ -187,7 +181,7 @@ matrix:
     - jdk: oraclejdk9
       env:
         - DESC="build with JDK9"
-        - CMD="mvn -e package -Passembly"
+        - CMD="mvn -e package"
```

Comment:
what is a reason ?

Reply:
To make sure that all the tests have passed. `-Passembly` was a workaround at a time when Powermock was incompatible with JDK9. Now it is even possible to build the `site` phase.
I'm going to carefully check the compatibility with JDK9, after completing this task.